mobile-widgets/qml/main.qml

```diff
@@ -19,6 +19,7 @@ Kirigami.ApplicationWindow {
 		preferredHeight: Math.round(Kirigami.Units.gridUnit * (Qt.platform.os == "ios" ? 2 : 1.5))
 		maximumHeight: Kirigami.Units.gridUnit * 2
 		background: Rectangle { color: subsurfaceTheme.primaryColor }
+		backButtonEnabled: false
```

Comment:
i think this button is actually used.
would be great if others can confirm.

Reply:
I checked the current user guide and I don't see it there.

It's unaligned on current Kirigami and it takes even more space on newer Kirigami versions.

The button is so small on an iPhone 7 Plus, I doubt it's usable on smaller screens anyway.

Reply:
We do not use it on iOS, because swipe is active, but did you check if it is connected to an action on android (I am not on my mac the next hours so cannot really check).

I agree it is useless on small devices, provided there are e.g. swipe.

Reply:
To be sure it's clear which button is disabled here. This will just make it look like before.

Current master:
![back](https://user-images.githubusercontent.com/30494/39968148-d4f50c34-56c8-11e8-882b-701f48c73839.jpg)

From the user manual:
![user manual](https://subsurface-divelog.org/mobile-images/Editdive.jpg)

Reply:
i remember using this while testing on the desktop, but it looked fine.
perhaps we should fix the alignment instead, as i do think that folks actually use it.

waiting on more comments:
@dirkhh @atdotde